vpn/options.go

```diff
@@ -71,6 +74,12 @@ const (
 
 	// protoUDP is used for vpn in UDP mode.
 	protoUDP = proto("udp")
+
```

Comment:
please, create a new `const ( )` block for constants using a different type.

Reply:
done